src/components/Jobs/components/Job/Job.js

```diff
@@ -42,12 +75,74 @@ export class Job extends Component {
     return <span>{datePart} {timePart} {todayPart}</span>;
   }
 
+  createJobUrl = id => {
+    const { stripes: { okapi } } = this.props;
+
+    const { url: host } = okapi;
+
+    return createUrl(`${host}/change-manager/jobExecutions/${id}/records`);
+  };
+
+  createDeleteJobHeaders = () => {
+    const { stripes: { okapi } } = this.props;
+
+    return createOkapiHeaders(okapi);
+  };
```

Comment:
Are you sure you need this method? All it does is extract a prop and return another function, moreover, it's used only in `deleteJob` method.

Reply:
agree,  removed